pages/linux/pacdiff.md

```diff
@@ -0,0 +1,32 @@
+# pacdiff
+
+> Pacorig, pacnew and pacsave maintenance utility for pacman.
+> More information: <https://manned.org/pacdiff>.
+
+- Look for files that need maintenance:
+
+`pacdiff`
+
+- Use sudo and sudoedit to remove and merge files:
+
+`pacdiff --sudo`
+
+- Store the original version of any file you `(O)verwrite with` as a `.bak`:
```

Comment:
Two points on this one:

1. The `[o]verwrite` syntax is used to indicate that a given concept (overwrite) maps to an option. Since this doesn't have a `-o` option we don't have that mapping. Is there another reason you chose to use the `[o]verwrite` syntax?
2. As you mentioned in a [prior comment](https://github.com/tldr-pages/tldr/pull/8474#discussion_r965517546) overwriting only happens when the `-o` flag is passed; this example doesn't use that. Is this example description correct?

Reply:
1. That linked code isn't a flag, rather input given during program operation. You are prompted for each file found, and choose what to do with it. Entering `O` to overwrite is one of the options. I put it in graves because it's the exact prompt selection from the code. Is there a better way to specify it?

2. I was trying to clarify with this change that the `--backup` flag was only relevant when the `o` option was selected by using the exact text from the prompt.

Reply:
@adrienthebo this is the prompt given
```bash
ask "(V)iew, (M)erge, (S)kip, (R)emove %s, (O)verwrite with %s, (Q)uit: [v/m/s/r/o/q] " "$file_type" "$file_type"
```
How should I clarify that the flag `--backup` applies to when the input `O` is given for a file? Would not using a code block, and instead saying "(O)verwrite with" in quotes be better?

Reply:
> How should I clarify that the flag --backup applies to when the input O is given for a file? Would not using a code block, and instead saying "(O)verwrite with" in quotes be better?

It's undefined in our style guide how you should specify `O` in this case. Until it's formalized use what you and other reviewers prefer. Good catch - this should be described in our style guide. ;)

Reply:
> > How should I clarify that the flag --backup applies to when the input O is given for a file? Would not using a code block, and instead saying "(O)verwrite with" in quotes be better?
> 
> It's undefined in our style guide how you should specify `O` in this case. Until it's formalized use what you and other reviewers prefer. Good catch - this should be described in our style guide. ;)

Yeah, we should document this case, as some other programs might have prompts too.

Reply:
For the moment you run this example I think `current` is more accurate.
```suggestion
- Backup the current version of any file you `(O)verwrite` as a `.bak`:
```

Reply:
Also, if I'm explaining the full command behavior:
```suggestion
- Review files that need maintenance creating a backup (`.bak`) of the original version of any file you `(O)verwrite`:
```

Reply:
I think original is more clear then current...

Does the entire command functionality need to be reestablished on each explanation? I don't see this trend followed on other pages.

Reply:
I think the example description should explain all the command behavior. It isn't clear for me that this command starts a review process.

Reply:
Is the new language clear enough?

Reply:
`pacfile` is not domain language, if you google it you don't receive links to Arch Linux.

Reply:
Saying `.pac*` doesn't work either...

Reply:
is this revision good descriptive enough @navarroaxel?